docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -562,7 +562,7 @@ kind: query
 spec:
   name: Detect active processes with Log4j running
   platforms: macOS, Linux
-  description: Returns a list of active processes and the Jar paths which are using Log4j. Version numbers are usually within the Jar filename.
+  description: Returns a list of active processes and the Jar paths which are using Log4j. Version numbers are usually within the Jar filename. Note: This query is resource intensive and might kill machines with little or no Swaps.
```

Perhaps a bit more clear (and less scary) like this?

```suggestion
  description: Returns a list of active processes and the Jar paths which are using Log4j. Version numbers are usually within the Jar filename. Note: This query is resource intensive and has caused problems on systems with limited swap space. Test on some systems before running this widely.
```

---

Thanks, @zwass. This sounds way much better!!!